godtools/ViewControllers/Platform/GTLanguagesTableViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  GTLanguagesTableViewController.swift
+//  godtools
+//
+//  Created by Devserker on 4/19/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import UIKit
+
+protocol GTLanguagesTableViewControllerDelegate {
+}
+
+class GTLanguagesTableViewController: GTBaseViewController, UITableViewDataSource, UITableViewDelegate {
```

Comment:
I know why you did this for now so it's okay, but let's let the LanguagesManager be the delegates for the table.